roles/openshift_version/tasks/main.yml

```diff
@@ -86,8 +86,16 @@
   include: set_version_rpm.yml
   when: not is_containerized | bool
 
-- name: Set openshift_version for containerized installation
-  include: set_version_containerized.yml
+- block:
+  - name: Set openshift_version for containerized installation
+    include: set_version_containerized.yml
+  - name: Determine openshift rpm version
+    include: rpm_version.yml
+  - name: Fail if rpm version and docker image version are different
```

Comment:
I see the `main.yml` has additional 5 occurrences of the `fail`. So I guess this should be fixed with one huge `replace-fail-with-assert` PR.